src/main/scala/nl/biopet/tools/bamstats/generate/ArgsParser.scala

```diff
@@ -37,13 +37,36 @@ class ArgsParser(toolCommand: ToolCommand[Args])
   opt[File]('b', "bam") required () valueName "<file>" action { (x, c) =>
     c.copy(bamFile = x)
   } text "Input bam file"
-  opt[Int]("binSize") valueName "<int>" action { (x, c) =>
-    c.copy(binSize = x)
-  } text "Bin size of stats (beta)"
-  opt[Int]("threadBinSize") valueName "<int>" action { (x, c) =>
-    c.copy(threadBinSize = x)
-  } text "Size of region per thread"
+  opt[File]("bedFile") valueName "<file>" action { (x, c) =>
+    c.copy(bedFile = Some(x))
+  } text "Extract information for the regions specified in the bedfile."
+  opt[File]("excludePartialReads") action { (_, c) =>
+    c.copy(excludePartialReads = true)
+  } text "Exclude reads that originate from another region."
+  // How should this option be adequately named?
```

Comment:
Maybe just explicitly call this scatterMode. Current name does not specify that only the start is looked at.

Reply:
:+1: Great. I was looking for a name that would sufficiently describe this.